tests/test_photoCalib.py

```diff
@@ -33,23 +35,29 @@
 import lsst.pex.exceptions
 
 
-def computeMaggiesErr(instFluxErr, instFlux, calibrationErr, calibration, flux):
-    """Return the error on the flux (Maggies)."""
+def nJyToMagnitude(flux):
+    """Return an AB magnitude given a flux in nanojansky."""
+    return (flux*units.nJy).to(units.ABmag).value
+
```

Comment:
This routine is the only obvious missing piece.  I know it's defined in the header in utils, and it's here in the tests ... but it's not really transparent to the user on how to do this conversion in the stack.  That is, even a pure python pair of convenience routines that simply calls astropy (`nJyToMagnitude(flux)` and `abMagnitudeToNanojansky(mag)` or something, it's hard to do with the camel casing...) that are accessible from `lsst.afw.image` would be very useful.  These would replace what is currently there, `afwImage.abMagFromFlux()` and `afwImage.fluxFromABMag()` via `calib.cc`, which are unitless and confusing, but convenient.

Reply:
I was actually thinking to remove this method entirely and just use astropy directly in this test. The method only exists here because I originally wrote the math in it, before realizing I could just delegate it to astropy directly.

Reply:
So I can't reply to the comment below for some reason.  Anyway, I think this would be useful, if only because the way to do it is non-obvious (sorry astropy).  But since dm is only releasing fluxes, the common user will want to do this, yes.  But will they use the stack to do it is the relevant question.